src/main/java/lz/modelTesting/controllers/ProjectsController.java

```diff
@@ -58,14 +58,23 @@ public ProjectsController(ProjectsRepository projectsRepository, ObjectMapper ob
     @PostMapping(value = "/add")
     public ResponseEntity<String> addProject(HttpServletRequest request) {
         Project project = createProjectFromRequest(request);
+        projectsRepository.save(project);
         String id = project.getId();
         return ResponseEntity.ok().body(id);
     }
 
-    private Project createProjectFromRequest(HttpServletRequest request) {
+    @PostMapping(value = "/changename/{id}")
+    public ResponseEntity<String> changeNameOfProject(HttpServletRequest request,@PathVariable String id) {
         String name = request.getParameter(PROJECT_NAME);
         Project project = new Project(name);
+        project.setId(id);
         projectsRepository.save(project);
```

Comment:
It would be better to search project by id and then set the new name instead of creating new project